ansible/roles/collectory/tasks/main.yml

```diff
@@ -61,16 +65,17 @@
     - properties
 
 - name: copy all template configs
-  template: src={{item}} dest={{data_dir}}/collectory/{{item}}
+  template: src={{item}} dest={{data_dir}}/ala-collectory/{{item}}
   with_items:
-    - config/collectory-config.properties 
+    - config/ala-collectory-config.properties
+    - config/connection-profiles.json
```

Comment:
Looks like it exists in roles/collectory/files/data/config. Is it a template or a static file? If it's a template, please move it to the templates directory. If it is static, please remove the reference at line 71.